examples/jsm/exporters/GLTFExporter.js

```diff
@@ -1135,19 +1135,43 @@ class GLTFWriter {
 
 			}
 
-			pending.push( toBlobPromise.then( blob =>
+		}
+
+		if ( options.binary === true ) {
+
+			pending.push(
 
-				writer.processBufferViewImage( blob ).then( bufferViewIndex => {
+				toBlobPromise
+					.then( blob => writer.processBufferViewImage( blob ) )
+				     	.then( bufferViewIndex => {
```

Comment:
Why is there an extra indentation here? Also, seems like it's using soft tabs?

Reply:
I thought it was easier to read with the indentation, but I can remove it if you like.

Reply:
The indentation make is look as if it's inside a call back or a if statement.

Reply:
Hmm, maybe it was the soft tabs that made it confusing?

<img width="537" alt="Screen Shot 2022-05-10 at 3 55 13 PM" src="https://user-images.githubusercontent.com/97088/167566797-57503c81-de02-4ced-9dfd-d97dddb54737.png">

Reply:
I think the soft tabs were only on the `.then( bufferViewIndex => {` line. Should I keep the lines without identation or revert it back (but obviously fixing the soft tabs on that line)?

Reply:
Yeah, I think it was the soft tabs. I've fixed it.